utils/custom-lint-rules/use-ck-slash-imports.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Note: you need to run `yarn add -WD file:./utils/custom-lint-rules` after changes for them to be reflected locally
+ */
+
+/**
+ * Formatting a string array to an english conjuction
+ * @example
+ * ```js
+ * const topics = ["bears", "beets", "Battlestar Galactica"];
+ * formatter.format(topics)
+ * // => "bears, beets, and Battlestar Galactica"
+ * ```
+ * */
+const formatter = new Intl.ListFormat('en', { style: 'long', type: 'conjunction' });
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'encourage imports from canvas-kit subdirectories',
+      category: 'Possible Errors',
+      recommended: true,
+    },
+    fixable: 'code',
+  },
+  create: function(context) {
+    return {
+      ImportDeclaration(node) {
+        const {value} = node.source;
+        const {specifiers} = node;
+        const specifierNames = specifiers.map(specifier => {
+          if (specifier.type === 'ImportDefaultSpecifier' || specifier.type === 'ImportNamespaceSpecifier') {
+            return specifier.local.name;
+          }
+          if (specifier.type === 'ImportSpecifier') {
+            return specifier.imported.name;
+          }
+        });
+        const importSourceRegExp = /^@workday\/canvas-kit-(labs-|preview-)*react*$/;
+        if (importSourceRegExp.test(value)) {
+          const importsInEnglish = formatter.format(specifierNames);
+          const multipleSpecifierMessage = `Import ${importsInEnglish} from their Canvas Kit subdirectories instead of ${value}`;
```

Comment:
I'm channeling my inner Nicholas. I feel like the error message should also give you a way to solve this. 
`Import PrimaryButton from their Canvas Kit subdirectories. Example Import {PrimaryButton} from "@workday/canvas-kit-react" should be import {PrimaryButton} from "@workday/canvas-kit-react/button""`

I know this is super verbose but this leave no doubt as to what you have to do to fix the error?

Reply:
Yeah, I thought about that too. We would need to have an up-to-date map of where every import comes from. The closest thing we have is the V5 codemod, but that is now out of date.

Reply:
If there was a way to know what subdirectory to pull from, I'd happily add that.

Reply:
yeah that's fair, I was just thinking of an example text and not one that represents the import you're fixing which was hacky. Either way, big win